src/Actions/BcgovSettings.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+namespace Bcgov\Theme\Block\Actions;
+
+/**
+ * A settings page for the BCGov Block Theme, which registers the theme settings and allows users to modify them.
+ *
+ * @since 1.1.2
+ */
+class BcgovSettings {
```

Comment:
with the namespace of `Bcgov\Theme\Block\Actions` i'm wondering why you just didn't use 
```suggestion
class Settings {
```